README.md

```diff
@@ -2,14 +2,32 @@
 An Ansible playbook for installing and configuring MAAS, further documentation is found [here](https://maas.io/docs/ansible-playbooks-reference).
 
 ## Supported versions
-This playbook has been tested with Ansible version 5.10.0 and above. We recommend using the latest available stable version of Ansible (currently 7.x). The `netaddr` Python library needs to be installed on the machine on which Ansible is used; note that this is not required on remote hosts.
+This playbook has been tested with Ansible version 5.10.0 and above. We recommend using the latest available stable version of Ansible (currently 7.x). The `netaddr` Python library needs to be installed on the machine on which Ansible is used; note that this is not required on remote hosts. If you are running on Mac OS, you will also need to install the passlib python module.
+
 
 ## Install
 
 ```
 git clone git@github.com:maas/maas-ansible-playbook
 ```
 
+To install all python requirements (including Ansible itself) into a virtual Python 3 environment, you should navigate to the root of the repository cloned above and run:
+
+```bash
+python3 -m venv maas-ansible-venv
+source maas-ansible-venv/bin/activate
+pip install -r requirements.txt
```

Comment:
I would suggest `python3 -m pip install [stuff]` here

Reply:
What do you mean by `[stuff]` ? The idea behind this command ist to showcase how to install the full python environment from the requirements.txt file included in this repository.

Reply:
Replacing `pip install` with `python3 -m pip install` is what I was suggesting, to avoid issues with multiple pythons/pips/other mess that occasionally occurs on people's systems. Leaving the rest of the line as is.

Reply:
Ah ok, i like the idea to avoid any mess ups on peoples systems :)
I am not sure however if the additional `python3 -m pip install` is needed here, since the path of `pip` should be explicitly defined after sourcing (activating) the virtual environment. Is there a specific example where you have seen this to fail ?

Reply:
Not specifically in venv (not something I use often)- however using `python3 -m pip install` syntax is both consistent with use of pip outside of venv and consistent with the docs if I'm not mistaken- hence I prefer it over `pip install`